utils/generator/templates/component

```diff
@@ -1,24 +0,0 @@
-class {{ component.class_name }}({% if component.is_response %}JSONResponse{% else %}dict{% endif %}):
```

Comment:
Not doing anything with structured responses yet so deleting this for now. However structured responses can be directly added on to what we have without it being a breaking change (since structured responses will still be accessible like a dict/list)